src/test/java/me/nurio/events/PerformanceTest.java

```diff
@@ -1,23 +1,24 @@
 package me.nurio.events;
 
-import me.nurio.events.handler.Event;
-import me.nurio.events.handler.EventHandler;
-import me.nurio.events.handler.EventListener;
+import me.nurio.events.api.Event;
+import me.nurio.events.annotation.EventHandler;
+import me.nurio.events.api.EventListener;
 import me.nurio.events.internal.ReflectedEventManager;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
 
 /**
  * This class is used to manually perform fast benchmarking of the call event process.
  */
-@Ignore
-public class PerformanceTest {
+@Disabled
+ class PerformanceTest {
 
-    private EventManager eventManager = new ReflectedEventManager();
+    private final EventManager eventManager = new ReflectedEventManager();
```

Comment:
This is a disabled test, it's something internal. Also, there is a big debate about if using finals or not, so not always is a good idea to make it final.

If your main change doesn't need to change other things, why touch it? :)